src/app/(app)/missing-shoes/actions/get-missing-shoe-by-id.ts

```diff
@@ -0,0 +1,28 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+
+export const getMissingShoeById = async (requestId: number) => {
+  const supabase = await getSupabaseClient();
+  const { data: missingShoe, error } = await supabase
+    .from(Tables.MissingShoes)
+    .select()
+    .eq("id", requestId)
+    .maybeSingle();
+
+  if (error) {
+    throw error.message;
+  }
+
+  return {
+    id: missingShoe.id,
+    status: missingShoe.status,
+    ownerName: missingShoe.owner_name,
```

Comment:
@muhammad-bilal-ali-khokhar what's our protocol for DB properties? Do we store them all in snake case?

Reply:
Sir we are using snake case , but now i just check that some are in calmel case that is wrong I'll fix it today